extensions/amp-script/0.1/amp-script.js

```diff
@@ -669,7 +669,7 @@ export class SanitizerImpl {
     if (!this.allowForms_ && FORM_ELEMENTS.includes(tag)) {
       user().warn(
         TAG,
-        'Form elements (%s) are not allowed without sandbox="allow-forms".',
+        '[amp-script] Form elements (%s) cannot be mutated unless your <amp-script> includes the attribute sandbox="allow-forms".',
```

Comment:
I think the "[amp-script]" is duplicated (it's the same as TAG).

Reply:
Thanks for catching that. I was wondering why there was a `TAG` and a `tag`...